app/audit/models/CreditSummaryAuditing.scala

```diff
@@ -32,7 +34,7 @@ object CreditSummaryAuditing {
   }
 
   private def toDescription(creditType: CreditType)
-                           (implicit messages: MessagesApi): String = messages(creditType.key)(Lang("en"))
+                           (implicit messages: MessagesApi): String = messages(s"credit.description.${creditType.key}")(Lang("en"))
```

Comment:
:thinking: we might need to introduce concept of "default language" and apply it to audit only scope

Reply:
Lang was already there from messages API, could raise in deep dive